spec/models/agents/evernote_agent_spec.rb

```diff
@@ -0,0 +1,297 @@
+require 'spec_helper'
+
```

Comment:
Would it be worth extracting the nested Classes out of the EvernoteAgent into a Gem, or unit testing them here?